modules/RTC/JitsiRemoteTrack.js

```diff
@@ -64,26 +73,35 @@ export default class JitsiRemoteTrack extends JitsiTrack {
         this.muted = muted;
         this.isP2P = isP2P;
 
+        logger.debug(`New remote track added: ${this}`);
```

Comment:
please don't log full objects like this, mobile suffers a lot

Reply:
Hmm... Why? We have `toString`, isn't it used?

Reply:
Oh, we do? Then yeah, no worries!